src/Database/Driver/Mysql.php

```diff
@@ -230,21 +223,6 @@ public function supportsDynamicConstraints(): bool
         return true;
     }
 
-    /**
-     * Returns connected server version.
-     *
-     * @return string
-     */
-    public function getVersion(): string
```

Comment:
Is this a new method? Or are we breaking compatibility here?

Reply:
I split some internal version checks into a `getVersion()` helper in this commit https://github.com/cakephp/cakephp/commit/57848d731c87c03e04fa11f53810f961da3e8731 for 4.1.

Reply:
@markstory It's a new method, so there is no BC break. I went with `version()` instead of `getVersion()` as `Sqlserver` class already has it and there isn't going to be a setter for version so the "get" prefix isn't really necessary.